src/DataStructures/DataBox.hpp

```diff
@@ -780,14 +785,43 @@ SPECTRE_ALWAYS_INLINE constexpr void add_variables_compute_tags_to_box(
        '0')...});
 }
 
+template <
+    typename VariablesTag, typename... Ts, typename... Tags,
+    Requires<not tt::is_a_v<Variables, item_type<VariablesTag>>> = nullptr>
+SPECTRE_ALWAYS_INLINE constexpr void add_variables_item_tags_to_box(
+    databox_detail::TaggedDeferredTuple<Tags...>& /*data*/,
+    typelist<Ts...> /*meta*/) {}
+
+template <typename VariablesTag, typename... Ts, typename... Tags,
+          Requires<tt::is_a_v<Variables, item_type<VariablesTag>>> = nullptr>
+SPECTRE_ALWAYS_INLINE constexpr void add_variables_item_tags_to_box(
+    databox_detail::TaggedDeferredTuple<Tags...>& data,
+    typelist<Ts...> /*meta*/) {
+  const auto helper = [&data](auto tag_v) {
+    using tag = decltype(tag_v);
+    auto& vars = get<tag>(data.template get<VariablesTag>().mutate());
```

Comment:
Does the free `get` function work with TaggedDeferredTuple?

Reply:
I have two ways of interpreting your question, so I'll answer both cases:

1. TaggedDeferredTuple does not currently have a free get function because when I brought over the modified TaggedTuple I did not want to hunt through and replace all the `.template get` uses inside DataBox. This can be changed, but I think an issue should be filed and it should be done in a separate PR.

2. If you're confused about the outer `get` call, it is not on the TaggedDeferredTuple.

Let me know if this answers your question.

Reply:
Yeah the first interpretation is what I was intending. Sounds reasonable.

Reply:
Alright, made the issue: #346